src/XamlStyler.UnitTests/TestConfigurations/IndentWithTabsOverride.json

```diff
@@ -1,5 +1,6 @@
 {
   "IndentSize": 4,
   "IndentWithTabs": true,
-  "AttributesTolerance": 3
+  "AttributesTolerance": 3,
+  "IgnoreDesignTimeReferencePrefix": false
```

Comment:
Our documentation is not great in these test files -- for this one, we can leave the new option out.

Reply:
Corrected